pkgs/applications/editors/edbrowse/default.nix

```diff
@@ -2,7 +2,12 @@
 
 stdenv.mkDerivation rec {
   name = "edbrowse-${version}";
-  version = "3.7.4";
+  # Until edbrowse has a release containing
+  # https://github.com/CMB/edbrowse/commit/5d2b9e21fdf019f461ebe62738d615428d5db963
+  # we won't be able to link against the libcurl in nixpkgs.
+  # Once that happens, revert to using tarballs.
+  gitrev = "c32fdb0c12a16fc031fdfbf8171f8543b3b12e5a";
+  version = "git-${stdenv.lib.substring 0 7 gitrev}";
```

Comment:
`version` should be in the format of `unstable-YYYY-MM-DD`
> https://nixos.org/nixpkgs/manual/#sec-package-naming
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23"

Reply:
Fixed.